portal/views/user.py

```diff
@@ -1604,3 +1604,63 @@ def posted_filename(request):
         doc.uuid, user_id), user_id=current_user().id, subject_id=user_id,
         context='assessment')
     return jsonify(message="ok")
+
+@user_api.route('/user/<int:user_id>/password_reset', methods=('POST',))
+@oauth.require_oauth()
```

Comment:
I believe we want to restrict this to a set of roles.  Probably (and note you want it to be a list not a tuple, as a list get's any one, whereas tuple means all) [ROLE.INTERVENTION_STAFF, ROLE.STAFF, ROLE.ADMIN]